apps/transport/lib/datagouvfr/client.ex

```diff
@@ -22,6 +24,7 @@ defmodule Datagouvfr.Client do
       @spec post_process({:error, any} | {:ok, %{body: any, status_code: any}}) ::
               {:error, any} | {:ok, any}
       def post_process(response) do
+        Logger.debug fn -> "response: #{inspect(response)}" end
```

Comment:
do you want ot keep this log ?

Reply:
yep this is useful when you develop to know exactly what happened